components/Base/Input.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="my-4 flex flex-col">
+    <label
+      v-if="label"
+      class="mb-2"
+      :for="id"
+    >
+      {{ label }} {{ required ? '*' : '' }}
+    </label>
+    <input
+      v-bind="$attrs"
+      v-model="value"
+      class="border border-gray-300 rounded rounded-1 px-4 py-2 placeholder:font-light"
+    />
+  </div>
+</template>
+
```

Comment:
I left the BaseInput the way Marcin implemented it before. I think it needs improvements, but for now I wasn't focused on validation or anything like that. The goal of this PoC project is to explore supabase capabilities first, then deliver things that we are familiar with.